_i18n/ca/_posts/2019-09-25-ERP-per-a-cooperatives.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "ERP per a cooperatives: facilitant processos de gestió amb programari lliure"
+author: Sergi Alonso
+layout: post
+---
+
+Durant aquest any 2019 hem estat desenvolupant el projecte Singulars *“Estudi, desenvolupament i acompanyament per la implementació de sistemes ERP de programari lliure especialitzats en cooperatives i ESS”*. Hem pogut comprovar les necessitats de les entitats de l’Economía Social i Solidària i constatar que existeixen forces procesos susceptibles de ser millorats o implantats mitjançant eines TIC
+
+
+## Necessitat de millorar processos de gestió dins de les cooperatives
+
+Les cooperatives i altres entitats de l’economia social i solidària tenen necessitats de gestió equiparables a les de qualsevol altra organització:
+
+* Facturació
+* Comptabilitat
+* Gestió de projectes
+* Seguiment econòmic
+* Gestió de persones (persones sòcies treballadores i assalariades)
+* Gestió societària (persones sòcies de la cooperativa)
+* Gestió de la comunicació amb públic objectiu i partners
+* Altres activitats de l’empresa, tant primàries com secundàries
+
+Les cooperatives són entitats que conformen el nucli de l’Economia Social. Sorgeixen impulsades per donar resposta a necessitats socials posant especial atenció en el benestar de les persones i altres entitats implicades en el desenvolupament de la seva activitat: treballadores, proveïdores i clientes.
+
+Millorar processos de gestió i tenir més informació sobre l’estat de la cooperativa és un objectiu comú dins de l’ESS que permetrà prendre millors decisions per garantir la sostenibilitat econòmica i humana dins de la cooperativa. Per aquest motiu són necessàries propostes transversals que sorgeixin del cooperativisme.
+
+En aquest projecte Singulars, la feina conjunta realitzada entre **[FACTO](https://facto.cat/), [Som Connexió](https://somconnexio.coop/) i Coopdevs**, s’ha centrat a desenvolupar l’ERP Odoo perquè pugui ser implantat en qualsevol cooperativa, facilitant la instal·lació del programari i la documentació disponible de manera oberta per a qualsevol organització que en vulgui fer ús.
```

Comment:
Perhaps it would be good to add the link to Coopdevs too, in case someone copy-pastes or syndicates this content. (And because all other orgs mentioned already have it.)

Reply:
@naoisegolden  Both translations are in Catalan, at this moment we don't translate every Post